Ah, there are some issues on Windows. AppVeyor failed to compile:

```
..\Modules\signalmodule.c(219): error C2065: 'res': undeclared identifier [C:\projects\cpython\PCbuild\pythoncore.vcxproj]
..\Modules\signalmodule.c(219): warning C4047: '=': 'int' differs in levels of indirection from 'PyObject *' [C:\projects\cpython\PCbuild\pythoncore.vcxproj]
```

--- a/Doc/library/signal.rst
+++ b/Doc/library/signal.rst
@@ -341,7 +341,7 @@ The :mod:`signal` module defines the following functions:
       On Windows, the function now also supports socket handles.
 
    .. versionchanged:: 3.7
-      Added ``warn_on_full_buffer``.
+      Added ``warn_on_full_buffer`` parameter.
 
 .. function:: siginterrupt(signalnum, flag)
 

--- a/Lib/test/test_signal.py
+++ b/Lib/test/test_signal.py
@@ -92,9 +92,11 @@ class WindowsSignalTests(unittest.TestCase):
 class WakeupFDTests(unittest.TestCase):
 
     def test_invalid_call(self):
+        # First parameter is positional-only
         with self.assertRaises(TypeError):
             signal.set_wakeup_fd(signum=signal.SIGINT)
 
+        # warn_on_full_buffer is a keyword-only parameter
         with self.assertRaises(TypeError):
             signal.set_wakeup_fd(signal.SIGINT, False)
 

--- a/Modules/signalmodule.c
+++ b/Modules/signalmodule.c
@@ -216,7 +216,7 @@ report_wakeup_send_error(void* data)
     /* PyErr_SetExcFromWindowsErr() invokes FormatMessage() which
        recognizes the error codes used by both GetLastError() and
        WSAGetLastError */
-    res = PyErr_SetExcFromWindowsErr(PyExc_OSError, (int) (intptr_t) data);
+    PyErr_SetExcFromWindowsErr(PyExc_OSError, (int) (intptr_t) data);
     PySys_WriteStderr("Exception ignored when trying to send to the "
                       "signal wakeup fd:\n");
     PyErr_WriteUnraisable(NULL);

